docs/getting-started/dev-portal/introduction.md

```diff
@@ -1,16 +1,18 @@
 ## Introduction to Elements Dev Portal
 
-Elements Dev Portal is backed by the [Stoplight Platform](https://stoplight.io/) and can do many things. 
+Elements Dev Portal is backed by the [Stoplight Platform](https://stoplight.io/) and offers: 
 
-- Support multiple APIs
-- Markdown Articles
-- [Hosted Mock Server](https://meta.stoplight.io/docs/platform/3.-design/d.setting-up-a-mock-server.md) Integration
-- Version Selector (Multiple Branches)
-- Powerful Search
+- Support for multiple APIs
+- Markdown article rendering
+- [Hosted mock servers](https://meta.stoplight.io/docs/platform/3.-design/d.setting-up-a-mock-server.md) Integration
```

Comment:
I think the third one will display as 'Hosted mock servers Integration' - could just whack the Integration I think, or make it 'Hosted mock server integration'